be/src/storage/rowset/metadata_cache.h

```diff
@@ -48,6 +53,8 @@ class MetadataCache {
     void _erase(const std::string& key);
     static void _cache_value_deleter(const CacheKey& /*key*/, void* value);
 
+    static MetadataCache* _s_instance;
```

Comment:
shall we do it this way?

1. check TabletManager pointer nullable before retrieve the metadata cache
2. proactively clean the `DropTabletInfo` in ~TabletManager, or add `stop()` to clean it if it is too late in ~TabletManager.

Reply:
I think it's simple to do this, and same as `pagecache`.

Reply:
think of two points.
1. global instances have its pain, especially the global initialization sequence and destruction sequence which is hard to control. so less global instances is preferred whenever possible.
2. global instances doesn't guarantee to solve the exit problem. that was the biggest problem when @trueeyu fixed the BE exit issue, adding a lot of stop/exit interface to carefully craft the exit procedure.

So it would be good to eliminate global instance whenever possible as opposite a "simple" implementation but leave a lot of hidden works in future.

Reply:
I am good with this implementation for a quick fix of the crashing for the coming release.